src/lpython/tests/test_error_rendering.cpp

```diff
@@ -62,12 +62,19 @@ TEST_CASE("Error Render: primary/secondary labels, single line") {
     Location loc1, loc2, loc3;
     LocationManager lm;
     input = "One line text\n";
-    lm.in_filename = "input";
-    lm.get_newlines(input, lm.in_newlines);
-    lm.out_start.push_back(0);
-    lm.in_start.push_back(0);
-    lm.in_start.push_back(input.size());
-    lm.out_start.push_back(input.size());
+    {
+        std::ofstream out("input.txt");
+        out << input;
+        LocationManager::FileLocations fl;
+        fl.in_filename = "input.txt";
+        lm.get_newlines(input, fl.in_newlines);
```

Comment:
Here we write the `input` contents into the `input.txt`.
In the content `"One line text\n"`: the `\n` is changed to `\r\n`. So, the ctests fail, as \r is added up in the windows. I think, we either remove `\r` or handle this `test_error_rendering` separately for windows.
Error:
```
( semantic error: Error with label no message
 --> input.txt:1:5
  |
1 | One line text

  |     ^^^^ 
== semantic error: Error with label no message
 --> input.txt:1:5
  |
1 | One line text
  |     ^^^^ 
```

Reply:
I figured out, how to fix this...